src/components/data-management/SamplesTable.jsx

```diff
@@ -0,0 +1,120 @@
+/* eslint-disable import/no-unresolved */
+/* eslint-disable react/jsx-props-no-spreading */
+import React, { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Table, Row, Col } from 'antd';
+import { DEFAULT_NA } from 'redux/reducers/projects/initialState';
+import { updateExperiment } from 'redux/actions/experiments';
+import { updateProject } from 'redux/actions/projects';
+import arrayMove from 'array-move';
```

Comment:
Run

`npm update array-move`

then change import here to 

`import {arrayMoveImmutable} from 'array-move';`